docs/release-notes/Percona-Everest-1.8.0-(2025-06-29).md

```diff
@@ -0,0 +1,131 @@
+
+#### Key features
+
+- Import database backups from external storage, such as Amazon S3, directly into database clusters managed by Percona Everest.
+
+- Support various backup formats and tools, including logical and physical backups created using `pg_dump`,` mysqldump`, `mongodump`, and others.
+
+- A pluggable and extensible framework that can adapt to various import needs and workflows.
+    
+#### How to import external database backups using the Percona Everest UI
+
+Here are the steps to import the external backups:
+{.power-number}
+
+1. Navigate to the Percona Everest homepage and click **Import**.
+    
+2. Select the database type you want to import(MySQL, PostgreSQL, or MongoDB). The **Basic information** page will be displayed.
+
+3. Fill in the details on the **Basic information** page and click **Continue**. This will take you to the **Import information** page.
+
+4. On the **Import Information** page, enter the following:
+
+    - Choose the data importer from the dropdown.
+
+    - Provide S3 details.
+
+        ![!image](../images/mongodb_s3_details_importers.png)
+
+    -  Specify the **File Directory** path within your S3 bucket where the backup files are stored. Click **Save**.
+
+        Refer to the [documentation](https://docs.percona.com/everest/backups_and_restore/how_to_import_backups.html) for details on the correct file path format.
+
+        ![!image](../images/importers_mongo_file_path.png)
+
+    -  Enter the key-value pairs and user secrets (For MongoDB and MySQL) in the **DB credentials** section.
+
+        ![!image](../images/importers_mongodb_db_credentials.png)
+
+5. Click **Continue** until you reach the end of the wizard. You will be notified once the import is successfully completed.
+
+
+## New features
+
+- [EVEREST-2068](https://perconadev.atlassian.net/browse/EVEREST-2068), [EVEREST-2069](https://perconadev.atlassian.net/browse/EVEREST-2069), [EVEREST-2070](https://perconadev.atlassian.net/browse/EVEREST-2070): Starting with Percona Everest 1.8.0, you can now import external backups from PostgreSQL, MySQL, and MongoDB, respectively, into clusters managed by Percona Everest using a simplified and extensible DataImporter framework.
+
+
+
+## Improvements
+
+- [EVEREST-1806](https://perconadev.atlassian.net/browse/EVEREST-1806): We have improved the visibility of the proxy validation message when creating a MySQL database.
+
+- [EVEREST-1909](https://perconadev.atlassian.net/browse/EVEREST-1909): Duplicate IP/Netmask entries for external access are now prevented, ensuring accurate network configurations in database clusters.
+
+- [EVEREST-1946](https://perconadev.atlassian.net/browse/EVEREST-1946): When editing a database cluster, users might mistakenly enter an incorrect disk size. Since disk downscaling is not supported and resizing triggers a 6-hour lockout (e.g., for MongoDB on EKS), Percona Everest now prompts for confirmation before applying disk size changes.
```

Comment:
I don't think that this has anything to do with incorrect disk sizes, it mostly to warn users that this operation isn't reversible. I would also refrain from mentioning the 6-hour thing.


```suggestion
- [EVEREST-1946](https://perconadev.atlassian.net/browse/EVEREST-1946): Since disk resizing is an irreversible operation, Percona Everest now prompts for confirmation before applying disk size changes.
```